src/gsy_e/models/myco_matcher/__init__.py

```diff
@@ -1,9 +0,0 @@
-from gsy_e.models.myco_matcher.myco_internal_matcher import MycoInternalMatcher
```

Comment:
Can you please explain why you removed this module?
It helps to import the MatchingEngines simpler with
```
from gsy_e.models.myco_matcher import MatchingEngineExternalMatcher
```
Please also add the `MatchingEngineInternalForwardMarketMatcher` 
Thanks!